autogen/composio_autogen/autogen_toolspec.py

```diff
@@ -1,100 +1,16 @@
 import types
 import logging
 import hashlib
-from inspect import Parameter, Signature
-from typing import Union, List, Annotated
+from inspect import Signature
+from typing import Union, List
 
 import autogen
 from composio import ComposioCore, App, Action, FrameworkEnum
-from pydantic import create_model, Field
 from autogen.agentchat.conversable_agent import ConversableAgent
 
-logger = logging.getLogger(__name__)
-
-schema_type_python_type_dict = {
-    'string': str,
-    'number': float,
-    'boolean': bool,
-    'integer': int
-}
-
-fallback_values = {
-    'string': "",
-    'number': 0.0,
-    'integer': 0.0,
-    'boolean': False,
-    'object': {},
-    'array': []
-}
-
-def pydantic_model_from_param_schema(param_schema):

```

Comment:
This was not being used?

Reply:
Being used, but moved all these too the core, as shared.